yarn.lock

```diff
@@ -14114,6 +14129,13 @@ scheduler@^0.20.1, scheduler@^0.20.2:
     loose-envify "^1.1.0"
     object-assign "^4.1.1"
 
+scheduler@^0.23.0:
```

Comment:
The duplicate `scheduler` package is a bit scary. What's still using `scheduler@0.20.1`?

Reply:
I could track it back to `@eps1lon/enzyme-adapter-react-17`. I don't have a context about it. Do we still need it? Do we need a new one for React 18?

Reply:
I guess this adapter still works for React 18. Probably best to just fork it and adjust its dependency to use stuff from React 18.

Reply:
I've created a branch with the changes for the adapter for 18 - https://github.com/mnajdova/enzyme/tree/feat/publish-adapter-18 The `scheduler@0.20.1` is not gone, it came from the older version of `react-reconciler`. 

The last thing is not clear for me is https://github.com/mui/material-ui/pull/33196/files#diff-7293771b71fe2b08cf9ab5152009d5d1c387a6aa2440e68aba3977cc8ca2cebaR52